src/core/Table/TablePaginator.test.tsx

```diff
@@ -131,31 +154,6 @@ it('should render loading state when there is no data', () => {
   expect(container.querySelector('.iui-dropdown')).toBeFalsy();
 });
 
-it('should render disabled state when there is no data', () => {
```

Comment:
why removed this test?

Reply:
Because we are not showing pages list when there is no data.